modules/raft/src/integrationTest/java/org/apache/ignite/raft/jraft/core/ItNodeTest.java

```diff
@@ -2709,29 +2715,34 @@ public void testAppendEntriesWhenFollowerIsInErrorState() throws Exception {
             assertEquals(20, fsm.getLogs().size());
     }
 
-    @Test
-    @Disabled("https://issues.apache.org/jira/browse/IGNITE-21792")
+    @RetryingTest(maxAttempts = 2)
+    @Timeout(value = 25, unit = TimeUnit.SECONDS)
```

Comment:
Do I understand correctly that the test can hang for some reason now?

Reply:
I am not quite sure, I set the election timeout to 100 seconds later and I want to cancel the test before the timeout is reached - if the election timeout is triggered the followingTimes number isn't correct any more and the test fails.

Reply:
Why would you set the timeout to such a big value then? Why do we need two separate timeout mechanisms?

Reply:
I can set the timeout to 25_000 or 100_000_000 if that makes it clear that this timeout should not be reached.